### PR TITLE
Hyper-V: updated test cases priorities

### PR DESCRIPTION
**Pull request by chvalean**
Updated priorities for Hyper-V only test cases.
Shared Azure & Hyper-V tests have been already updated.

Separate features into individual xml files for:
- File copy (over vmbus)
- Production Checkpoint
- Runtime Memory Resize

### CODE REVIEW

**Review by lizzha — APPROVED** [review]

File: XML/TestCases/FunctionalTests-Network.xml [hunk 1]
```diff
@@ -123,7 +123,7 @@
         <Category>Functional</Category>
         <Area>NETWORK</Area>
         <Tags>network</Tags>
-        <Priority>2</Priority>
+        <Priority>3</Priority>
     </test>
     <test>
         <testName>NET-MAX-NICS</testName>
```
Comment (iamshital) [inline XML/TestCases/FunctionalTests-Network.xml:129]: Can we assign priority 0 for this test?

**Review by iamshital — DISMISSED** [review]

File: XML/TestCases/FunctionalTests-Network.xml (hunk 1 above)
Comment (juhlee-microsoft) [reply XML/TestCases/FunctionalTests-Network.xml:129]: Based on Azure update, it should be 1 not 0.

**Review by juhlee-microsoft — APPROVED** [review]
Please change the priority of Net-Max-NICs

File: XML/TestCases/FunctionalTests-Network.xml (hunk 1 above)
Comment (chvalean) [reply XML/TestCases/FunctionalTests-Network.xml:129]: Please check the entire xml to see the right priority:
- net max LEGACY NICs - this one has been moved as P3, we're talking about the actual hyper-v legacy networks.
- net max NICs - this one is already and remains as P1.

@juhlee-microsoft please dismiss Shital's review and merge this if @iamshital is off for the day.

**Pull request merged**